.gitignore

```diff
@@ -8,3 +8,4 @@ coverage.txt
 .editorconfig
 .bashrc
 .idea
+.DS_Store
```

Comment:
`*/.DS_Store` is better for any dir in the root. WDTY?

Reply:
@fuweid Actually .DS_Store already ignored all the files in sub directories, if this is what you mean.

Reply:
👌 LGTM